exploration/percent-format.md

```diff
@@ -162,14 +222,17 @@ You saved {$savings :percent} on your order today!
 > @sffc suggested that we should consider other names for `:percent`.
 > The name shown here could be considered a placeholder pending other suggestions.
 
-**Pros**
+_Pros_
 - Least verbose placeholder
-- Clear what the placeholder does; self-documenting?
+- Clear what the placeholder does; self-documenting
+- Consistent with separating `:currency`
```

Comment:
I don't see this consistency. If it applies to `:percent`, does that it also applies to `:permille`? How about `:meter` or `:radian`?

`:currency` makes sense to keep separate because currency formatting depends on currency codes, which in turn impact default fraction digits, and other presentation details. Nothing like that applies to percentage formatting.